types/stripe/stripe-tests.ts

```diff
@@ -1256,6 +1256,38 @@ stripe.accounts.createLoginLink('acct_17wV8KBoqMA9o2xk', 'http://localhost:3000'
     const created: number = loginLink.created;
     const url: string = loginLink.url;
 });
+
```

Comment:
The account persons tests should probably be another section to make it clear to future contributors.